roxctl/common/report/report.go

```diff
@@ -70,6 +70,24 @@ func JSON(output io.Writer, alerts []*storage.Alert) error {
 	return nil
 }
 
+// JSONRemarks pipes out the violated alerts and remarks as JSON.
+func JSONRemarks(output io.Writer, alerts []*storage.Alert, remarks []*v1.DeployDetectionRemark) error {
```

Comment:
Would this be a bit better name for that func, maybe?
 
```suggestion
func WriteJSONRemarks(output io.Writer, alerts []*storage.Alert, remarks []*v1.DeployDetectionRemark) error {
```

Reply:
This function supersedes `JSON` in some scenarios, which is why I tried to keep the name similar - but I've changed it to `JSONWithRemarks` to make it better understandable what's happening,